apps/backoffice-v2/src/common/hooks/useZodSearchParams/utils/deep-null-to-string.ts

```diff
@@ -0,0 +1,15 @@
+export const deepNullToString = (value: unknown) => {
```

Comment:
1. I think our convention is to start the function name with `convert`
2. Lodash has a convention to add the `deep` term as a suffix - decide whatever you think it's better. `deepNull` sounds very weird to me
3. A JSDoc `@example` annotation with array/object could be excellent here
4. Unit tests are necessary here.

```suggestion
export const convertNullToStringDeep = (value: unknown) => {
```